scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -674,35 +676,7 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-secondary">Pa
               <td>{{gene.description|title}}</td>
               <td>{{gene.region_annotation}}</td>
               <td>
-                {% set n_primary_transcripts = [] %}
```

Comment:
Moving this to a single macro, located in scout/server/blueprints/cases/templates/cases/utils.html